src/scripts.js

```diff
@@ -31,28 +25,84 @@ let tripNumTravelersInput = document.getElementById('num-trav-input');
 let tripDestinationSelection = document.getElementById('destination-selection');
 let submitTripButton = document.getElementById('submitTrip');
 let estimatedCost = document.getElementById('estCost');
+let dashboard = document.getElementById('dashboard');
+let tripFormWrapper = document.getElementById('formWrapper');
+let loginPage = document.getElementById('loginPage');
+let loginForm = document.getElementById('login-form');
```

Comment:
There's a few instances here where your id's are both in camelCase and kebab-case to refactor whenever you have a minute for consistency!